config/blazer.yml

```diff
@@ -0,0 +1,73 @@
+# see https://github.com/ankane/blazer for more info
+
+data_sources:
+  main:
+    url: <%= ENV["BLAZER_DATABASE_URL"] %>
+
+    # statement timeout, in seconds
+    # none by default
+    timeout: 60
+
+    # caching settings
+    # can greatly improve speed
+    # off by default
+    cache:
+      mode: slow # or all
+      expires_in: 60 # min
+      slow_threshold: 15 # sec, only used in slow mode
+
+    # wrap queries in a transaction for safety
+    # not necessary if you use a read-only user
+    # true by default
+    # use_transaction: false
```

Comment:
We are using a read-only user in production but I figured using transactions still wouldn't hurt. If there is some significant downside to them please let me know!